serverless.yml

```diff
@@ -9,14 +9,15 @@ package:
 custom:
   esbuild:
     bundle: true
-    minify: true
+    minify: ${strToBool(${env:MINIFY, 'true'})}
     target: node16
+    sourcemap: ${strToBool(${env:SOURCEMAP, 'false'})}
```

Comment:
Wouldn't we want the default to be true here?

Reply:
As far as I'm aware of, `sourcemap` is `false` by default in our current template as this option was not there in our esbuild plugin config (not sure why). Therefore, I use an env variable to enable it for debugging.

Reply:
@TheOrangePuff I think we want sourcemaps turned on by default don't we? I'm not sure how much size it adds to the AWS deployment. May be worth doing as a separate change.

Reply:
I found an article that has some comparison between enabled/disabled source map: https://dev.to/aws-builders/how-to-use-source-maps-in-typescript-lambda-functions-with-benchmarks-4bo4 
Not sure how we would like it to be in the future but for now, it's disabled. Let's do it as a separate change.